runtime/test-runtime/src/lib.rs

```diff
@@ -711,6 +713,38 @@ sp_api::impl_runtime_apis! {
 		}
 	}
 
+	impl beefy_primitives::BeefyApi<Block, BeefyId> for Runtime {
```

Comment:
AFAICT the goal of `test-runtime` is to test node-runtime integrations, since that's something that is already tested in BEEFY repository I'n not sure what would be the goal here.
In case there is a test that we would like to write with this it can be implemented later on, but now I think it would just be unnecessary code to maintain.